src/main/java/net/snowflake/spark/snowflake/parquet/FileFromStream.java

```diff
@@ -0,0 +1,31 @@
+package net.snowflake.spark.snowflake.parquet;
```

Comment:
This work should be on a separate branch; we should merge the changes from the parquet feature branch and this version upgrade separately. Can we separate these commits?